agent-bootstrap/src/main/java/io/opentelemetry/auto/config/Config.java

```diff
@@ -329,15 +329,13 @@ public static String getSettingFromEnvironment(final String name, final String d
    * Calls {@link #getSettingFromEnvironment(String, String)} and converts the result to a list by
    * splitting on `,`.
    */
-  @NonNull
   private static List<String> getListSettingFromEnvironment(
       final String name, final String defaultValue) {
-    return parseList(getSettingFromEnvironment(name, defaultValue));
+    return Objects.requireNonNull(parseList(getSettingFromEnvironment(name, defaultValue)));
```

Comment:
`requireNonNull` shouldn't be used with return statements, it's really for validating parameters. I understand it's to replace the usage of `@lombok.NonNull` but I think we can just delete it here, especially since they're all private methods. Best would be for us to consistently add `@Nullable` and enable `NullAway` (not for this PR).

Reply:
Agree, I'll remove them.